homeassistant/components/yale_smart_alarm/coordinator.py

```diff
@@ -135,9 +135,7 @@ def get_updates(self) -> dict:
                 )
             except AuthenticationError as error:
                 raise ConfigEntryAuthFailed from error
-            except requests.HTTPError as error:
-                if error.response.status_code == 401:
-                    raise ConfigEntryAuthFailed from error
```

Comment:
Does this seem like an odd move?

This breaks re-auth flows?

Reply:
It's now handled in the upstream package and raises `AuthenticationError` which is on the lines above. So no longer a need to take care of it here.